.env.test

```
@@ -1,6 +1,6 @@
 NEXT_PUBLIC_GOOGLE_ANALYTICS_ID=GA_TEST_1234567890
-NEXT_PUBLIC_REOWN_PROJECT_ID=TEST_1234567890
+NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=TEST_1234567890
 ENVIRONMENT=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ENVIRONMENT key should go before the NEXT_PUBLIC_GOOGLE_ANALYTICS_ID key

(UnorderedKey)